ModelBundle/Document/FieldType.php

```diff
@@ -262,5 +262,6 @@ public function getTranslatedProperties()
     public function __clone()
     {
         $this->labels = new ArrayCollection();
+        $this->options = new ArrayCollection();
```

Comment:
you are duplicating those lines. Can you extract the collection initialization in a method which is called in both the contructor and clone method?